dotnetcore2.0/run/MockBootstraps/Program.cs

```diff
@@ -94,7 +151,7 @@ static string GetFunctionHandler(string[] args)
         static string GetEventBody(string[] args)
         {
             return args.Length > 1 ? args[1] : (Environment.GetEnvironmentVariable("AWS_LAMBDA_EVENT_BODY") ??
-              (Environment.GetEnvironmentVariable("DOCKER_LAMBDA_USE_STDIN") != null ? Console.In.ReadToEnd() : "{}"));
```

Comment:
Actually, if you try to supply no arguments with the current version JSON parsing will fail. With "Value should be quoted exception".

That is reproduced on my Windows machine. I will try Mac now.

Reply:
That doesn't make sense – doesn't that mean that `Console.In.ReadToEnd()` must also be surrounded by single quotes? That can't be right

Reply:
![docker](https://user-images.githubusercontent.com/34073648/49390476-bb24a800-f731-11e8-99bd-0efe95dfdfb3.gif)

That is why I've fixed that. But again maybe on Linux and Mac it works just fine I'm checking

Reply:
That error (I believe) is talking about double quotes around strings – ie, to parse a string as JSON, it needs to be surrounded by double quotes, ie: `"Hello World"`

But this is not a string, it's an empty object

Reply:
And this fix prevents this exception anyway. And I see just the same behavior on Mac.

@mhart does it make sense?

Reply:
I think you're misunderstanding me – of course it's a string, you can't parse anything else. What I'm saying is that the error you're seeing there is referring to parsing JSON strings – not JSON objects.

If you convert the string `Hello World` to JSON, it becomes `"Hello World"` – those are the actual characters in the string. So on the command line, or in JavaScript, you could represent that as `'"Hello World"'`. In .NET you could represent it like `"\"Hello World\""`

Reply:
With objects, or anything else, you shouldn't surround them with quotes in the string itself.

You can't do this:

```js
JSON.parse("'{}'")
```

If you want an empty object, you do this:

```js
JSON.parse("{}") // or JSON.parse('{}')
```

Reply:
Oh, yes, know I understood you! I think Newtonsoft serializer is capable of understanding single quoted Json strings and that did confuse you.

To avoid such confusion I'll replace them with double quotes, okay?

Reply:
It really shouldn't change. Go to `examples/dotnetcore2.1` and run the following:

```
docker pull lambci/lambda:dotnetcore2.1
docker run --rm -v "$PWD"/pub:/var/task lambci/lambda:dotnetcore2.1 test::test.Function::FunctionHandler
```

Reply:
You'll see that it parses the default empty object string just fine and outputs it on the command line.

```
START RequestId: a3c22922-440c-440c-afd9-7bd955590462 Version: $LATEST
inputEvent: {}
...
```

Reply:
(btw there's no such thing as single-quoted JSON strings – I'm not sure what you're referring to there?)

Reply:
Oh, well we have some misunderstanding again 😄 
Yes, it works for your example function, because it receives `object` as an input.

But my sample function receives `string` as an input, and therefore fails to execute.

```C#
/// <summary>
        /// A simple function that takes a string and does a ToUpper
        /// </summary>
        /// <param name="input"></param>
        /// <param name="context"></param>
        /// <returns></returns>
        public string Handler(string input, ILambdaContext context)
        {
            return input?.ToUpper();
        }
```

With my changes both work just fine 👍 

@mhart does it now make sense ?

Reply:
What!? Well of course it's failing then.

The default is an empty object – not a string!

Reply:
![issue1](https://user-images.githubusercontent.com/34073648/49392819-38531b80-f738-11e8-8dfc-ebb59e8caf9f.gif)

That is why I refer to "single quoted JSON strings" because Newtonsoft.Json understands them. And Lambda serializers internally uses them to convert streams

Reply:
@mhart and as for the default, why couldn't we provide default which will effectively work for both "HelloWorld" functions? I don't see anything wrong with that

Reply:
I can assure you this is not JSON 😄 http://json.org/

Reply:
@mhart I agree, but the fact remains fact it worked, that is why we had so much of a confusion here. 
Can file up an issue on Newtonsoft repo 😸

Reply:
And I will change the quotes to double

Reply:
@ndobryanskyy please leave the default as is – most Lambda functions that ppl write take an object of some sort – eg, an API Gateway event, an S3 event, etc. The web console also provides test events as obejcts. And all the other language runners in `docker-lambda` also default to an empty object. A string would be unusual – especially the string `'{}'` – that is *very* unusual and most confusing to ppl. Even the string `Hello World` would be more appropriate if it was going to be a string.

Reply:
@mhart, no problem, I will revert it, do you have any more comments around PR?